docs/api/wrapper/setSelected.md

```diff
@@ -1,9 +1,15 @@
-## setSelected(value)
+## setSelected()
 
-Sets a specified `<option>` as selected in a `<select>`.
+Selects a option element.
```

Comment:
Selects a option element.

`Selects a option element and updates v-model bound data.`